src/erc3525/extensions/slotenumerable/module.cairo

```diff
@@ -0,0 +1,166 @@
+#[starknet::contract]
+mod ERC3525 {
+    use starknet::{get_caller_address, ContractAddress};
+    use traits::Into;
+    use zeroable::Zeroable;
+    use integer::BoundedInt;
+
+    use cairo_erc_3525::src5::module::SRC5;
+    use cairo_erc_3525::erc721::module::ERC721;
+    use cairo_erc_3525::erc3525::module::ERC3525;
+    use cairo_erc_3525::erc3525::extensions::slotenumerable::interface::{IERC3525SlotEnumerable, IERC3525_SLOT_ENUMERABLE_ID};
+
+    #[storage]
+    struct Storage {
+        _slots_len: u256,
```

Comment:
No need for the `_` prefix for storage names here.
However, should we prefix with the contract name in order to avoid storage clashing?

Reply:
I use the `_` by convention, meaning these are protected/private features. This is my interpretation of what I different places, such as:
- https://github.com/ruleslabs/marketplace/blob/main/src/access/ownable.cairo#L23
- https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc721/erc721.cairo#L24

Regarding storage key clash, I was thinkg that it was not a problem anymore with the new syntax (thanks to packaging I guess) but I'm asking: https://t.me/sncorestars/29024